src/modules/dashboard/lists/ListItemWrapper.tsx

```diff
@@ -53,7 +54,10 @@ export const ListItemWrapper = ({
             </Typography>
           </Tooltip>
         </Link>
-        {frozen && <FrozenTooltip symbol={symbol} currentMarket={currentMarket} />}
+        {frozen && symbol != 'renFIL' && (
+          <FrozenTooltip symbol={symbol} currentMarket={currentMarket} />
+        )}
+        {frozen && symbol == 'renFIL' && <RenFILToolTip />}
```

Comment:
Should this be `!frozen`? It currently does not show on the dashboard since the asset is not frozen.

There's an active aip to freeze this asset, so maybe we just remove the frozen check completely so it shows now and continues to show after the aip is executed?

Reply:
disregard... it's working